sentry/src/main/java/io/sentry/DefaultSentryClientFactory.java

```diff
@@ -53,6 +53,14 @@
      * Default timeout of an HTTP connection to Sentry.
      */
     public static final int TIMEOUT_DEFAULT = (int) TimeUnit.SECONDS.toMillis(1);
+    /**
+     * Option to set a read timeout for requests to the Sentry server, in milliseconds.
+     */
+    public static final String READ_TIMEOUT_OPTION = "readtimeout";
+    /**
+     * Default read timeout of an HTTP request to Sentry.
+     */
+    public static final int READ_TIMEOUT_DEFAULT = (int) TimeUnit.SECONDS.toMillis(5);
```

Comment:
What is the default of `HttpConnection` when not read timeout is set?
It should be the same value here so we're not changing the behavior.

Reply:
I see below says default is `0`. 
I'd say then we should have `0` here too.

Reply:
Actually it's 5 seconds

Reply:
See `HttpConnection`

Reply:
Where did you see the 0 seconds?

Reply:
There's a comment on this file: `timeout New timeout to set. If 0 is used (java default) wait forever.`
But I see what you mean now, it's indeed 5 seconds. My bad,.